cvp_mesh_planner/src/cvp_mesh_planner.cpp

```diff
@@ -105,8 +105,7 @@ uint32_t CVPMeshPlanner::makePlan(const geometry_msgs::msg::PoseStamped& start,
 
     geometry_msgs::msg::PoseStamped pose;
     pose.header = header;
-    pose.pose = mesh_map::calculatePoseFromPosition(vec, goal_vec, face_normals[fH], dir_length);
-    cost += dir_length;
```

Comment:
I think we should continue update the cost of the path with the last bit of the path, e.g. like so:
```
cost += (goal_vec - vec).length();
```

Reply:
I have looked into this and after the loop in line 95 `vec` is the last entry of the path vector, which is the goal vector itself.  Therefore `(goal_vec - vec ).length()` will always be zero and we can skip this step.